### PR TITLE
Final bringup

### PR DESCRIPTION
**Pull request by jr1221**
## Changes

_Explanation of changes goes here_

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes # (issue #)

### CODE REVIEW

**Review by caiodasilva2005 — DISMISSED** [review]

File: Core/Src/dti.c [hunk 1]
```diff
@@ -14,13 +14,14 @@
...
 #include <string.h>
+#include <stdio.h>
 
 #include "bms.h"
 #include "can_handler.h"
 #include "emrax.h"
 
 #define CAN_QUEUE_SIZE 5 /* messages */
-#define SAMPLES	       20
+#define SAMPLES	       3
```
Comment (caiodasilva2005) [inline Core/Src/dti.c]: Verify number of samples for proper deadzone

File: Core/Src/pdu.c [hunk 2]
```diff
@@ -68,6 +75,87 @@ int init_ina(pdu_t *pdu, ina226_t *ina, uint16_t dev_addr, float r_shunt,
 	return 0;
 }
 
+bool verify_tca_config(pdu_t *pdu)
```
Comment (caiodasilva2005) [inline Core/Src/pdu.c:80]: verify if this is actually working and doing what we want it to do

File: Core/Src/pedals.c [hunk 3]
```diff
@@ -428,44 +469,61 @@ void vProcessPedals(void *pv_params)
...
-		uint16_t accel2_norm = pedal_percent_pressed(
-			accel2_volts, APPS2_VOLTAGE_OFFSET, MAX_APPS2_VOLTS);
+		float accel1_norm = pedal_percent_pressed(
+			accel1_volts, MIN_APPS1_VOLTS, MAX_APPS1_VOLTS);
+		float accel2_norm = pedal_percent_pressed(
+			accel2_volts, MIN_APPS2_VOLTS, MAX_APPS2_VOLTS);
 
-		calc_pedal_faults(accel1_volts, accel2_volts, accel1_norm,
-				  accel2_norm);
+		bool possible_faults = calc_pedal_faults(
```
Comment (caiodasilva2005) [inline Core/Src/pedals.c:494]: TODO: create an error handling standard. We do nothing with a lot of our errors possible_faults is an example of this

File: Core/Src/pedals.c [hunk 4]
```diff
@@ -428,44 +469,61 @@ void vProcessPedals(void *pv_params)
...
+		osMutexAcquire(brake_state_mut, osWaitForever);
+		if (brake_value > PEDAL_BRAKE_THRESH) {
+			brake_pressed = true;
+		} else {
+			brake_pressed = false;
+		}
+		osMutexRelease(brake_state_mut);
+		write_brakelight(pdu, brake_pressed);
 
 		if (calc_bspd_prefault(accel_value, brake_value)) {
```
Comment (caiodasilva2005) [inline Core/Src/pedals.c:518]: TODO: once proper noise filtering is set up on pedal lines then this can be properly tested and implemented for fsae

**Review by caiodasilva2005 — APPROVED** [review]
